packages/config/package.json

```diff
@@ -8,8 +8,8 @@
   "scripts": {
     "build-prod": "tsc || echo 'built, with errors'",
     "check-ts": "tsc --noEmit",
-    "clean-deps": "rm -rf node_modules",
-    "clean": "rm -f ./src/*.js ./src/**/*.js ./src/**/**/*.js ./test/**/*.js || echo 'cleaned'",
+    "clean-deps": "rimraf node_modules",
+    "clean": "rimraf ./src/*.js ./src/**/*.js ./src/**/**/*.js ./test/**/*.js || echo 'cleaned'",
```

Comment:
These clean commands that delete implementation files are wild; are they actively used?

Reply:
It seems those are actively used - the screenshot is running `yarn` in `10.0-release` branch

![image](https://user-images.githubusercontent.com/15656860/161169281-789bc726-3df0-41b6-8ca6-660746753312.png)

Reply:
Yeah, we don't use dist folders at cypress to avoid navigating to type files when changing packages. source maps would be a solution, but it does not seem to be one we considered for dev. we prefer using ts-node.

Reply:
When we run build-prod, on local, it will create a bunch of files that need to be cleaned up.